django/core/cache/backends/base.py

```diff
@@ -208,6 +208,9 @@ def set_many(self, data, timeout=DEFAULT_TIMEOUT, version=None):
 
         If timeout is given, that timeout will be used for the key; otherwise
         the default cache timeout will be used.
+
+        Returns a list of keys that were not inserted, returns None if all keys
```

Comment:
It seems a bit confusing to have this docstring here without any `return` statement,  perhaps some clarification would be helpful.